packages/daemon/src/host.js

```diff
@@ -282,23 +313,26 @@ export const makeHostMaker = ({
             return formulaIdentifier;
           }
 
-          const { formulaIdentifier: lookupFormulaIdentifier } =
-            await provideLookupFormula(petNamePath);
-          return lookupFormulaIdentifier;
-        }),
+          // TODO:lookup Check if a formula already exists for the path. May have to be
+          // done in the daemon itself.
+          return petNamePath;
```

Comment:
This function appears to either return a pet name path or a formula identifier. Is a “pointer” the union of pet name path and identifier? If so, how are they distinguished?

Reply:
I'm renaming the property / array to `endowmentFormulaIdsOrPaths`. In this particular array, strings are formula identifiers and arrays are pet name paths. They are distinguished using `typeof value === 'string'`.

Reply:
Ah, the plural agreement is confusing, but there’s perhaps nothing for it. I would like to consider (in follow-up) whether we can normalize these to formula identifiers up front.